addons/project/static/tests/tours/project_update_tour_tests.js

```diff
@@ -150,10 +150,10 @@ tour.register('project_update_tour', {
     trigger: ".o_field_widget[name=description] div[name='milestone'] ul li:contains('(12/12/2099 => 12/12/2100)')",
     run: function () {},
 }, {
-    trigger: ".o_field_widget[name=description] div[name='milestone'] ul li span:contains('(due 12/12/2022)')",
+    trigger: ".o_field_widget[name=description] div[name='milestone'] ul li:contains('(due 12/12/2022)')",
     run: function () {},
 }, {
-    trigger: ".o_field_widget[name=description] div[name='milestone'] ul li span:contains('(due 12/12/2100)')",
+    trigger: ".o_field_widget[name=description] div[name='milestone'] ul li:contains('(due 12/12/2100)')",
```

Comment:
It doesn't look like the fact that these were in spans was essential but best to make sure.

cc @dmo-odoo

Reply:
This is the template that adds them:
```xml
<ul class="o_checklist" t-if="milestones['list']">
<t t-foreach="milestones['list']" t-as="milestone" t-key="milestone['id']">
<li t-attf-class="{{milestone['is_reached'] and 'o_checked' or ''}}">
<t t-esc="milestone['name']"/>
<span t-if="milestone['is_deadline_future'] and not milestone['is_reached']"><font style="color: rgb(190, 190, 190);"><t t-set="color_level" t-value="64"/><t t-call="project.milestone_deadline"/></font></span>
<span t-else=""><t t-set="color_level" t-value="128"/><t t-call="project.milestone_deadline"/></span>
</li>
</t>
</ul>
```

Reply:
This is the template that adds them (at `project/views/project_update_templates.xml`):
```xml
<ul class="o_checklist" t-if="milestones['list']">
<t t-foreach="milestones['list']" t-as="milestone" t-key="milestone['id']">
<li t-attf-class="{{milestone['is_reached'] and 'o_checked' or ''}}">
<t t-esc="milestone['name']"/>
<span t-if="milestone['is_deadline_future'] and not milestone['is_reached']"><font style="color: rgb(190, 190, 190);"><t t-set="color_level" t-value="64"/><t t-call="project.milestone_deadline"/></font></span>
<span t-else=""><t t-set="color_level" t-value="128"/><t t-call="project.milestone_deadline"/></span>
</li>
</t>
</ul>
```
@tlibioulle Could this `span` just as well have been a `t`? Is it important the span remains? With the last PR of this commit, if the `span` is in the editor and it has no attributes, its contents are unwrapped. Wanna make sure there's no implications for this feature.

Reply:
Hello @Zinston ,

FYI, Thibault is no longer at Odoo.

I quickly tested it on your runbot and it seems to be rendered as before.
However, I am not sure to understand how it will be rendered if we let the span. 🤔

Also, if you don't know how to test that part then don't hesitate to ask to my team, Services Team (RD Fun Vidange on discord) or my PO (Alexandra (AJU)) 🙂

You can also ask to Alexandra to test that part once your development will be finished. 🙂

Reply:
in fact, now, the span, it does not take into account, right? 🤔

Reply:
Thanks, will do. What do you mean it does not take the span into account?
It would be a really strange case if we had a span without styling or classes that was anyway necessary or was targeted in css.